app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,37 @@
+class CommentsController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    post = Post.find(params[:post_id])
+    render json: post.comments
+  end
+
+  def new
+    @comment = Comment.new
+    render :new
+  end
+
+  def create
+    comment_author = current_user
+    post = Post.find(params[:post_id])
+    post_author = post.user
+
+    comment = params[:comment]
+
+    new_comment = Comment.new(comment.permit(:text))
```

Comment:
- [ ] As mentioned above, let's extract the permit to a private method. You can learn more aout the strong parameters in this [lesson](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/lessons/Controllers_lesson.md#:~:text=Pay%20special%20attention%20to%20the%20strong%20parameters%20description%20in%20Rails%20Guides)
```ruby
  private 
  def comment_params
    params.require(:comment).permit(:text)
  end
```

Reply:
- [ ] Kindly use comment params  method and make it private.